PackageHandling/Install-AzDevops.ps1

```diff
@@ -14,12 +14,14 @@ try {
     Try {
         $az_upgrade = Write-Output n | az upgrade 2>&1
```

Comment:
Why not just run 
![image](https://github.com/microsoft/navcontainerhelper/assets/10775043/d079021c-8d39-4bfd-9805-7482d240d0e6)
and then if this fails (i.e. AZ is not installed), then do the install below=

Reply:
Because that comes with a UI:-)

Reply:
Yes and az version doesn't tell me if there is a new version. 
This is the only trick i've found to check if an update is available.